tests/v1/kv_connector/unit/test_nixl_connector.py

```diff
@@ -703,7 +794,7 @@ def test_kv_connector_stats_aggregation():
 
     # Create KVOutputAggregator for 3 workers (simulating TP=3), same thing
     # done in MultiprocExecutor.execute_model
-    aggregator = KVOutputAggregator(world_size=3)
+    aggregator = KVOutputAggregator(expected_finished_count=3)
```

Comment:
ignore, to be rebased once https://github.com/vllm-project/vllm/pull/26734 lands